app/(groups)/_layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { Stack } from 'expo-router';
+import React from 'react';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+export default function GroupsLayout() {
+  return (
+    <SafeAreaView style={{ flex: 1 }}>
```

Comment:
Why we use style ? We can't use class name ?

Reply:
fixed